src/Fable.Import.RemoteDev.fs

```diff
@@ -29,8 +29,10 @@ module RemoteDev =
     type ExtensionOptions
         [<ParamObject; Emit("$0")>]
         (
-            name: string
+            ?getActionType: obj -> obj,
+            ?name: string
         ) =
+        member val getActionType: obj -> obj = jsNative with get, set
         member val name: string = jsNative with get, set
```

Comment:
Should be updating to `'T option` for consistency

Reply:
Done.